server/sonar-server/src/main/java/org/sonar/server/permission/index/PermissionIndexerDao.java

```diff
@@ -118,7 +118,9 @@ public boolean isAllowAnyone() {
     "      FROM projects " +
     "      INNER JOIN user_roles ON user_roles.resource_id = projects.id AND user_roles.role = 'user' " +
     "      WHERE " +
-    "        (projects.qualifier = 'TRK' or  projects.qualifier = 'VW') " +
+    "        (projects.qualifier = 'TRK' " +
+    "         or  projects.qualifier = 'VW' " +
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "         or  projects.qualifier = 'VW' " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1192)